### ISSUE TITLE
[BUG] Postgres variables

### ISSUE THREAD

**Issue by Renato66** [cweijan/vscode-mysql#644]
I'm trying to use [variables in postgres](https://www.geeksforgeeks.org/postgresql-variables/)

```sql
DO $$ 
DECLARE
   counter    INTEGER := 1;
   first_name VARCHAR(50) := 'John';
   last_name  VARCHAR(50) := 'Doe';
   payment    NUMERIC(11,2) := 20.5;
BEGIN 
   RAISE NOTICE '% % % has been paid % USD', counter, first_name, last_name, payment;
END $$;
``` 

error:
``` 
SQL:  DO $$ DECLARE counter INTEGER := 1

unterminated dollar-quoted string at or near "$$ DECLARE counter INTEGER := 1"
```

- OS: Linux Ubunto
- Database Type: PostgresSql

**Comment by cweijan** [comment]
Thanks for your feedback, I will fix it in the next version.

**Comment by cweijan** [comment]
Hi, if you run all SQL it can run successfully, the program can't detect all SQL statements yet.
![image](https://user-images.githubusercontent.com/27798227/180645181-69b74641-3f40-458c-aee6-8be9aa691f88.png)

**Comment by cweijan** [comment]
Hi, I have released version 5.7.6, which supports do syntax.

**Comment by Renato66** [comment]
Thanks 🙏
